src/containers/App/App.js

```diff
@@ -123,11 +123,11 @@ class App extends React.Component {
         />
         
         <div className="wrapper">
-          <aside className={`${isMenuOpen ? 'open' : 'closed'}`}>
+          <aside>
             <Menu isAuthenticated={isAuthenticated} />
           </aside>
     
-          <section>
+          <section  className={`${isMenuOpen ? 'open' : 'closed'}`}>
             <div  
```

Comment:
@ceanela This button needs to be hooked up to close the menu when clicked.

Reply:
I'll take a look